app/controllers/admin/dashboard_controller.rb

```diff
@@ -17,7 +17,8 @@ def index
     @statsdrafts = Article.drafts.where('created_at > ?', today).count
     @statspages = Page.where('published_at > ?', today).count
     @statuses = Note.where('published_at > ?', today).count
-    @statuserposts = Article.published.where('published_at > ?', today).count(conditions: { user_id: current_user.id })
+    # app/controllers/admin/dashboard_controller.rb#L20
```

Comment:
This is the fix that Nathan posted to for the label error that denied access to admin